code-submissions-clustering-plugin/build.gradle.kts

```diff
@@ -41,4 +41,17 @@ tasks {
             }
         }
     }
+    register("calculate-dist", BaseCLITask::class) {
+        dependsOn(build)
+        val input: String? by project
+        val output: String? by project
+        args = mutableListOf<String>().apply {
+            add("calculate-dist")
+            input?.let { add("--input_file=$it") }
+            output?.let { add("--output_path=$it") }
+            if (project.hasProperty("serialize")) {
```

Comment:
I think we should also add the saveCSV flag. And you have almost the same tasks, maybe can we unify them?